src/categorical_algebra/FinCats.jl

```diff
@@ -406,20 +416,41 @@ not be possible.
 
 See also: [`is_natural`](@ref).
 """
-function is_functorial(F::FinDomFunctor; check_equations::Bool=false)
+function is_functorial(F::FinDomFunctor; return_failures::Bool=false, 
```

Comment:
Unfortunately this now conflicts with #762, so you'll have to rebase to use that API.